### PR TITLE
e2e: Convert the virtual workspace test to shared fixture

### PR DESCRIPTION
**Pull request by marun**

### CODE REVIEW

File: test/e2e/virtual/workspaces/virtual_workspace_test.go [hunk 1]
```diff
@@ -313,15 +322,15 @@ func testWorkspacesVirtualWorkspaces(t *testing.T, standalone bool) {
 				require.NoError(t, err, "failed to create root workspace roles")
 
 				err = server.virtualWorkspaceExpectations[0](func(w *tenancyv1beta1.WorkspaceList) error {
-					expectedOrgs := sets.NewString(orgName)
+					expectedOrgs := sets.NewString(orgName, "default")
```
Comment (marun) [inline test/e2e/virtual/workspaces/virtual_workspace_test.go:325]: @davidfestal I needed this change to get the tests passing, but I'm assuming this is not desirable. Any idea what I might have done and how to fix it?

Comment (sttts) [reply test/e2e/virtual/workspaces/virtual_workspace_test.go:325]: welcome to shared test data (users with groups) 🎉

Workaround: one group membership per test that depends on it.

Comment (sttts) [reply test/e2e/virtual/workspaces/virtual_workspace_test.go:325]: actually even per-test users.

Comment (marun) [reply test/e2e/virtual/workspaces/virtual_workspace_test.go]: One test did require a unique group to ensure auth isolation. I updated a different test that was using "root:default" to use a test org instead. PTAL

**Review by sttts — APPROVED** [review]

**Pull request merged**